packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -150,7 +150,6 @@ export type LocalTracks = {
     audio?: LocalSystemAudioTrack;
     video?: LocalDisplayTrack;
   };
-  annotationInfo?: AnnotationInfo;
```

Comment:
also the import of `AnnotationInfo` can be removed now, right?

Reply:
Remove it